ansible/psi/README.md

```diff
@@ -0,0 +1,36 @@
+Resources for setting up CI in PSI OpenStack
+============================================
+
+We have a "front-door-ci" project on the PSI OpenStack clouds rhos-d and rhos-01. The latter's nested KVM support and performance is good enough for our tasks runners.
+
+Access
+------
+Sign into the OpenShift web console on https://api.rhos-01.prod.psi.rdu2.redhat.com with group "redhat.com", your RedHat user name, and your Kerberos password. You need to be in the [front-door-ci group](https://rover.redhat.com/groups/group/front-door-ci) for this.
+
+The only change that was, and currently needs to be done manually is to add SSH and ICMP to the "default" security group. This step may be automated in the future.
+
+Automation setup
+----------------
+
+You need to install the OpenStack SDK, either with `sudo dnf install python3-openstacksdk` (on Fedora), or `pip install openstacksdk`.
+
+After that, you need to set up `~./config/openstack/clouds.yml`. If you don't already have one, copy or symlink [clouds.yml](./clouds.yml), otherwise merge it with your existing one. You need to add your credentials to `~/.config/openstack/secure.yaml`, see the comment in [clouds.yml](./clouds.yml) for how to do that.
+
+Check that this works with
+
+    openstack --os-cloud rhos-01 server list -n
+    ansible-inventory -i inventory/openstack.yml -v --yaml --list
```

Comment:
```
Using /etc/ansible/ansible.cfg as config file
[WARNING]: Couldn't list Openstack hosts. See logs for details
Invalid input for field 'identity/password/user/password': None is not of type 'string'

Failed validating 'type' in schema['properties']['identity']['properties']['password']['properties']['user']['properties']['password']:
    {'type': 'string'}

On instance['identity']['password']['user']['password']:
    None (HTTP 400) (Request-ID: req-90b8c377-8894-4d07-962c-9c47aa0c46f8)
all:
  children:
    ungrouped: {}
```

Do I need to log in? First command worked with `--os-username mmarusak` and then passing password